galaxy/Dockerfile

```diff
@@ -176,12 +176,11 @@ RUN mkdir -p /shed_tools $GALAXY_CONFIG_TUS_UPLOAD_STORE \
     && chown $GALAXY_USER:$GALAXY_USER /shed_tools $GALAXY_CONFIG_TUS_UPLOAD_STORE \
     && ln -s /tool_deps/ $EXPORT_DIR/tool_deps \
     # Configure Galaxy to use the Tool Shed
-    && chown $GALAXY_USER:$GALAXY_USER $EXPORT_DIR/tool_deps \
+    && chown $GALAXY_USER:$GALAXY_USER $EXPORT_DIR/tool_deps $GALAXY_VIRTUAL_ENV -R \
```

Comment:
The venv had files owned by root and that caused problems for me. I have no idea why this only happens for me and not for you and the CI.

Reply:
The flower installation just before this task installs packages into that directory. I think that was the issue. I hadn’t built the image locally since the last few refactorings, but now, when I do, I see the problem. Surprisingly, the build was successful in the CI environment. Thanks for fixing it!

Reply:
Can we convince the step before to run the pip install step as `galaxy` user?

Reply:
There is no option at the [task](https://github.com/usegalaxy-eu/flower-ansible-role/blob/19e75b0876d59713fcd339064bbbf33d4716e7e3/tasks/main.yml#L58) level. Since some other tasks require root privileges, I can't set the user at play level.